### PR TITLE
Add Papr support

### PR DESCRIPTION
**Pull request by rhatdan**
papr support copied in and modified from buildah

### CODE REVIEW

**Comment by cevich** [comment]
@mheon @rhatdan I made some comments https://github.com/projectatomic/libpod/pull/5#pullrequestreview-73608149 that got lost.  May be relevant here, I dunno.

File: .papr.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+branches:
+  - master
+
+host:
+  distro: fedora/26/atomic
```
Comment (cevich) [inline .papr.yml]:
```yaml
host:
  distro: fedora/26/atomic
  specs:
    ram: 4096
```

**Comment by rhatdan** [comment]
@jlebon Any idea why papr is not running?

File: .papr.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -xeuo pipefail
+
+export GOPATH=$HOME/gopath
+export PATH=$HOME/gopath/bin:$PATH
+export GOSRC=$HOME/gopath/src/github.com/projectatomic/libpod
+
+(mkdir -p $GOSRC && cd /code && cp -r . $GOSRC)
+
+dnf install -y \
```
Comment (cgwalters) [inline .papr.sh]: See also https://github.com/ostreedev/ostree/blob/master/ci/libbuild.sh - once something is packaged then you can just do `pkg_install_builddeps libpod`.

File: .papr.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+branches:
+  - master
```
Comment (cgwalters) [inline .papr.yml]: I'd still have `auto` and `try` here.

**Comment by rhatdan** [comment]
@cgwalters Any ideas why the  papr tests are not running now?

**Comment by rhatdan** [comment]
@jlebon Looks like I hung up papr again?  Or it is just slow?

**Comment by jlebon** [comment]
@rhatdan It's just slow :(. Push notifications aren't going through currently, so it can be 5-10 minutes before the next poll picks it up.

**Comment by rhatdan** [comment]
@jlebon Just tell me to "cool my jets" thanks.

**Comment by rhatdan** [comment]
@cevich @mheon @baude PTAL

**Comment by baude** [comment]
LGTM

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
@umohnani8 PTAL

**Comment by mheon** [comment]
Tests are green, LGTM!

**Pull request merged**